### ISSUE TITLE
Provide Block.txt docs

### ISSUE THREAD

**Issue by matteomallus** [OneBusAway/onebusaway-gtfs-modules#145]
**Summary:** 

GtfsReader currently reads Block (block.txt), whose all fields are required (block_seq_num,block_var_num,block_route_num,block_run_num) but I can't manage to find any online documentation about the meaning of these fields, which are not part of the GTFS standard.

I'm interested in using this feature, and some docs about the meaning of the fields would be helpful. Can you please provide some explanation about the meaning of these fields?

**Steps to reproduce:** 

While browsing the online documentation I can't find any reference about the meaning of the following fields: block_seq_num,block_var_num,block_route_num,block_run_num

**Expected behavior:** 

The Wiki should provide some explanation, of the non-standard fields 

**Observed behavior:** 

An explanation of these fields cannot be found.

**Platform:** 

-

**Comment by barbeau** [comment]
@matteomallus I'm not sure there is broad agreement on what this file should look like.

For example, DART publishes this file but with different fields:
https://transitfeeds.com/p/dart/26/20190807/file/blocks.txt

```
SERVICE_ID,BLOCK_ID,PULLOUTTIME,PULLINTIME
2,201,15360,70380
2,202,18000,67740
2,203,19500,34200
2,204,22020,34140
```

**Comment by barbeau** [comment]
It's possible this was added to OBA to support a specific producer, but it would only be useful in the context of that producer (and I don't know who that is).

**Comment by matteomallus** [comment]
@barbeau  Ok, thanks.

I found a GTFS which uses these fields in 
https://openmobilitydata.org/p/king-county-metro/73/20190830/file/block.txt

But I can't find an explanation. Thank you anyway!

**Comment by barbeau** [comment]
@matteomallus yes, given the origin of OneBusAway that would make sense. You might want to take a look at KCM or Sound Transits websites to see if they have an explanation. Or reach out it the open data folks there.
